packages/fx-core/src/plugins/resource/frontend/plugin.ts

```diff
@@ -173,12 +174,22 @@ export class FrontendPluginImpl {
       };
     }
 
-    const authPlugin = ctx.configOfOtherPlugins.get(DependentPluginInfo.RuntimePluginName);
-    if (authPlugin) {
-      runtimeEnv = {
-        endpoint: authPlugin.get(DependentPluginInfo.RuntimeEndpoint) as string,
-        startLoginPageUrl: DependentPluginInfo.StartLoginPageURL,
-      };
+    if (isArmSupportEnabled()) {
```

Comment:
we need to enable static web app feature in post provision, when arm support enabled